pkgs/kde/frameworks/kidletime/default.nix

```diff
@@ -0,0 +1,13 @@
+{
+  mkKdeDerivation,
+  qtwayland,
+  pkg-config,
+  wayland-protocols,
+  xorg,
+}:
+mkKdeDerivation {
+  pname = "kidletime";
+
+  extraNativeBuildInputs = [pkg-config];
+  extraBuildInputs = [qtwayland xorg.libXScrnSaver wayland-protocols];
```

Comment:
I am surprised to use xorg here, I thought they went wayland first and only.

Reply:
First, but not only. The X11 session is still supported, though not recommended.